src/site/blocks/alert.drupal.liquid

```diff
@@ -34,7 +34,9 @@
     {% if expander.fieldTextExpander == empty %}
         {{ expander.fieldWysiwyg.processed }}
     {% endif %}
-
+    {% comment %} <div id="react-applicationStatus" class="static-page-widget" data-widget-type="Form-2346">
+      <a class="usa-button-primary va-button-primary" href="/mdt/2346">Order Hearing Aid, Batteries, and Accessories</a>
+    </div> {% endcomment %}
```

Comment:
I know you did this for testing and visibility into your work, but I don't think this should be merged to master. It's totally fine locally though. Can we exclude this from your PR?

Reply:
Ok I will take that out.

Reply:
That changed has been pushed.